src/interfaces/javaapi/javaapi.go

```diff
@@ -109,57 +145,41 @@ func (r ResponseService) GetTarget() string {
 	return r.RemoteTargetInfo.Target
 }
 
-const logPrefix = "interface"
-
-// Handle Platform Dependencies
-const (
-	platform      = "android"
-	executionType = "android"
-
-	logStr          = "/log"
-	configStr       = "/apps"
-	dbStr           = "/data/db"
-	certificateFile = "/data/cert"
-
-	cipherKeyFile = "/user/orchestration_userID.txt"
-	deviceIDFile  = "/device/orchestration_deviceID.txt"
-)
-
-var orcheEngine orchestrationapi.Orche
-
-var (
-	edgeDir             string
-	logPath             string
-	configPath          string
-	dbPath              string
-	certificateFilePath string
-	cipherKeyFilePath   string
-	deviceIDFilePath    string
-)
-
 // ExecuteCallback is required to launch application in java layer
 type ExecuteCallback interface {
 	androidexecutor.ExecuteCallback
 }
 
+var (
+	commitID, version, buildTime string
+	buildTags                    string
+)
+
 // OrchestrationInit runs orchestration service and discovers remote orchestration services
 func OrchestrationInit(executeCallback ExecuteCallback, edgeDir string) (errCode int) {
+	initPlatformPath(edgeDir)
 
 	logmgr.Init(logPath)
 	log.Printf("[%s] OrchestrationInit", logPrefix)
-
-	logPath = edgeDir + logStr
-	configPath = edgeDir + configStr
-	dbPath = edgeDir + dbStr
-	certificateFilePath = edgeDir + certificateFile
-
-	cipherKeyFilePath = edgeDir + cipherKeyFile
-	deviceIDFilePath = edgeDir + deviceIDFile
+	log.Println(">>> commitID  : ", commitID)
```

Comment:
This build tags will be **empty**, if we generate a binding for java/android, that is why we removed earlier.

Reply:
this log is removed on additional patch.